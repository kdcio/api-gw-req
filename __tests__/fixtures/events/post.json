{
  "httpMethod": "POST",
  "body": "{\"message\":\"hello world\"}"
}
