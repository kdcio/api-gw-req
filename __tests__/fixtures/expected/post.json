{
  "path": "",
  "method": "POST",
  "query": {},
  "headers": {},
  "ip": "",
  "userAgent": "",
  "pathParams": "",
  "body": { "message": "hello world" },
  "authorizer": {}
}
